src/renderer/components/+extensions/extensions.tsx

```diff
@@ -38,6 +39,21 @@ interface InstallRequestValidated extends InstallRequestPreloaded {
   tempFile: string; // temp system path to packed extension for unpacking
 }
 
+interface ExtensionState {
+  displayName: string;
+  // Possible states the extension can be
+  state: "uninstalling";
+}
+
+/**
+ * Extensions that were removed from extensions but are still in "uninstalling" state
+ */
+function getRemovedUninstalling(extensions: InstalledExtension[], extensionState: ObservableMap<string, ExtensionState>) {
+  return Array.from(extensionState.entries()).filter(([id, extension]) => 
+    extension.state === "uninstalling" && !extensions.find(extension => extension.id === id)
+  ).map(([id, extension]) => ({ ...extension, id }));
+}
```

Comment:
I think this would be better as a `@computed get removedUninstalling() { ... }` on the `Extensions` class bellow.

Reply:
Converted